Dockerfile

```
@@ -93,7 +94,8 @@ RUN set -ex \
   && cd ./sopel-src \
   && su-exec sopel python setup.py install --user \
   && cd .. \
-  && rm -rf ./sopel-src
+  && rm -rf ./sopel-src \
+  && apk del gcc build-base
```

```suggestion
  && apk del .build-base
```
see comment above about virtual packages.